src/main/webapp/profile.js

```diff
@@ -42,6 +51,20 @@ function submitProfileForm(methodType, redirectUrl) {
     });
 }
 
+function possibleQualifiedJobs() {
```

Comment:
Better just make a class and document.getByClassName()

Reply:
Done